app/src/main/java/com/bunbeauty/papakarlo/feature/auth/LoginFragment.kt

```diff
@@ -0,0 +1,177 @@
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.focus.FocusRequester
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextRange
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.input.TextFieldValue
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.navigation.fragment.findNavController
+import androidx.navigation.fragment.navArgs
+import com.bunbeauty.papakarlo.R
+import com.bunbeauty.papakarlo.common.BaseComposeFragment
+import com.bunbeauty.papakarlo.common.extension.navigateSafe
+import com.bunbeauty.papakarlo.common.ui.element.FoodDeliveryScaffold
+import com.bunbeauty.papakarlo.common.ui.element.button.LoadingButton
+import com.bunbeauty.papakarlo.common.ui.element.textfield.FoodDeliveryTextField
+import com.bunbeauty.papakarlo.common.ui.theme.FoodDeliveryTheme
+import com.bunbeauty.papakarlo.feature.main.IMessageHost
+import com.bunbeauty.shared.presentation.login.Login
+import com.bunbeauty.shared.presentation.login.LoginViewModel
+import org.koin.androidx.viewmodel.ext.android.viewModel
+
+class LoginFragment : BaseComposeFragment<Login.State, Login.Action, Login.Event>() {
+
+    override val viewModel: LoginViewModel by viewModel()
+
+    private val args: LoginFragmentArgs by navArgs()
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        viewModel.handleAction(Login.Action.Init)
+    }
+
+    @Composable
+    override fun Screen(state: Login.State, onAction: (Login.Action) -> Unit) {
+        FoodDeliveryScaffold(
+            backActionClick = {
+                onAction(Login.Action.BackClick)
+            },
+            backgroundColor = FoodDeliveryTheme.colors.mainColors.surface,
+            actionButton = {
+                LoadingButton(
+                    modifier = Modifier
+                        .padding(horizontal = FoodDeliveryTheme.dimensions.mediumSpace),
+                    textStringId = R.string.action_login_continue,
+                    isLoading = state.isLoading,
+                    onClick = {
+                        onAction(Login.Action.NextClick)
+                    }
+                )
+            }
+        ) {
+            Column(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .verticalScroll(rememberScrollState())
+                    .padding(FoodDeliveryTheme.dimensions.mediumSpace),
+                horizontalAlignment = Alignment.CenterHorizontally,
+                verticalArrangement = Arrangement.Center
+            ) {
+                BoxWithConstraints {
+                    if (maxHeight > 200.dp) {
```

Comment:
200 lets move to cons like MAX_LOGO_SIZE?

Reply:
For Compose conts we should use `MaxLogoSize`

Reply:
fixed